bdio-tinkerpop/src/main/java/com/blackducksoftware/bdio2/tinkerpop/spi/BlackDuckIoReaderSpi.java

```diff
@@ -135,7 +135,16 @@ protected void getNodeProperties(Map<String, Object> node, BiConsumer<Object, Ob
     protected void getNodeEdges(Map<String, Object> node, BiConsumer<String, Object> edges) {
         node.forEach((term, value) -> {
             if (frame().isObjectProperty(term)) {
-                edges.accept(term, frame().context().fromFieldValue(term, value));
+
```

Comment:
Given we have very little knowledge of testing in tinkerpop, are there tests we can put into place to exercise this.   Additionally, can you give more context in regards to this change?